wagtailvideos/models.py

```diff
@@ -236,9 +236,7 @@ def get_transcode(self, media_format):
         except Transcode.DoesNotExist:
             return self.do_transcode(media_format)
 
-    def video_tag(self, attrs):
-        kw_attrs = {}
-
+    def video_tag(self, attrs, kw_attrs={}):
```

Comment:
Having a mutable default argument in a function can lead to surprising bad things: http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments. Better to set it to None, and then do an `if None: kw_attrs={}` in the function body.